src/server/controllers/Grblhal/GrblHalController.js

```diff
@@ -479,7 +480,8 @@ class GrblHalController {
                     line = line.replace(`${tool?.[0]}`, `(${tool?.[0]})`);
                 }
 
-                return line;
+                // remove all comments before forwarding on
+                return line.replace(/([^\)]*)/gm, '').replace(commentMatcher, '');
```

Comment:
Change this to return line.replace(/\([^\)]*\)/gm, '').trim();

Reply:
https://github.com/Sienci-Labs/gsender/pull/448/commits/c609723ecffe93c6ce104906b192a6f8be2ffdea